R/bootstrap.R

```diff
@@ -1013,7 +1013,10 @@ buildTabItem <- function(index, tabsetId, foundSelected, tabs = NULL,
 
 buildNavItem <- function(divTag, tabsetId, index) {
   id <- paste("tab", tabsetId, index, sep = "-")
-  title <- tagGetAttribute(divTag, "title")
+  # Get title attribute directory (not via tagGetAttribute()) so that contents
+  # don't get passed to as.character().
+  # https://github.com/rstudio/shiny/issues/3352
+  title <- divTag$attribs$title
   value <- tagGetAttribute(divTag, "data-value")
```

Comment:
Do you know if this was also changed (looking now)?

Reply:
Yea looks like it https://github.com/rstudio/shiny/pull/3315/files#diff-befe9146792f0d65aeeaf781e8ba00a1a3638ac2be162b61b6c8958440860b56L986